source/IOSResolver/src/IOSResolverSettingsDialog.cs

```diff
@@ -248,6 +262,33 @@ private class Settings {
                 }
             }
 
+            if (settings.podfileAddUseFrameworks) {
+                GUILayout.BeginHorizontal();
+                GUILayout.Label("Enable Swift Framework Support Workaround",
+                    EditorStyles.boldLabel);
+                settings.swiftFrameworkSupportWorkaroundEnabled =
+                    EditorGUILayout.Toggle(settings.swiftFrameworkSupportWorkaroundEnabled);
+                GUILayout.EndHorizontal();
+                GUILayout.Label("This workround patches the Xcode project to properly link Swift " +
+                                "Standard Library when some plugins depend on Swift Framework " +
+                                "pods by:");
+                GUILayout.Label("1. Add a dummy Swift file to Xcode project.");
+                GUILayout.Label("2. Enable 'CLANG_ENABLE_MODULES' and " +
+                                "'ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES' build settings and set " +
+                                "'SWIFT_VERSION' to the value below.");
+
+                if (settings.swiftFrameworkSupportWorkaroundEnabled) {
+                    GUILayout.BeginHorizontal();
+                    GUILayout.Label("Swift Framework Version",
+                        EditorStyles.boldLabel);
+                    settings.swiftLanguageVersion =
+                        EditorGUILayout.TextField(settings.swiftLanguageVersion);
+                    GUILayout.EndHorizontal();
+                    GUILayout.Label("Used to set 'SWIFT_VERSION' build setting in Xcode. Leave " +
+                                    "it blank to update it manually.");
```

Comment:
Rather than saying "update it manually", would it be more accurate to say something like "use the "Swift Language Version" build property in the Xcode project"?

I ask because it isn't clear to me what is meant by updating it manually.

Reply:
Updated